### PR TITLE
[RISCV] Update immediate branching extension

### PR DESCRIPTION
**Pull request by melonedo**
Spec: https://github.com/openhwgroup/cv32e40p/blob/62bec66b36182215e18c9cf10f723567e23878e9/docs/source/instruction_set_extensions.rst#immediate-branching-operations

Update the format and the command line flag of the immediate branching extension.

### CODE REVIEW

File: llvm/test/CodeGen/RISCV/corev/bi.ll [hunk 1]
```diff
@@ -0,0 +1,41 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -O0 -mtriple=riscv32 -mattr=+m -mattr=+xcvbi -verify-machineinstrs < %s \
```
Comment (jeremybennett) [inline llvm/test/CodeGen/RISCV/corev/bi.ll]: We don't need the `+mattr=+m` to turn on the RISC-V mutliply extension.

**Review by jeremybennett — CHANGES_REQUESTED** [review]
One small change needed, then this can be merged.

File: llvm/test/CodeGen/RISCV/corev/bi.ll (hunk 1 above)
Comment (melonedo) [reply llvm/test/CodeGen/RISCV/corev/bi.ll]: Fixed

**Comment by jeremybennett** [comment]
This is now good to merge.

**Pull request merged**